packages/xdg_directories/test/xdg_directories_test.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path/path.dart' as path;
+import 'package:mockito/mockito.dart' show Fake;
+import 'package:process/process.dart';
+
+import 'package:xdg_directories/xdg_directories.dart' as xdg;
+
+void main() {
+  final Map<String, String> fakeEnv = <String, String>{};
+  Directory tmpDir;
+  setUp(() {
+    tmpDir = Directory.systemTemp.createTempSync('xdg_test');
+    fakeEnv.clear();
+    fakeEnv['HOME'] = tmpDir.path;
+    fakeEnv['XDG_CACHE_HOME'] = path.join(tmpDir.path, '.test_cache');
```

Comment:
Since the `path.join(tmpDir.path, foo);` construction is used everywhere in this test, I think it would be more readable if you made a helper like `testPath(String foo)` to call everywhere.

Reply:
OK, done.